README.md

```diff
@@ -58,34 +65,48 @@ echo 'export PATH="/usr/local/opt/ruby/bin:$PATH"' >> ~/.bash_profile
 
 # Install Bundler and dependencies
 gem install bundler
-bundle install --path vendor/bundle
+bundle config set path './vendor/cache'
+bundle install
 npm i babel-minify
 ```
 
-## Running locally :running:
+## Building :running:
+
+Docker (Windows/Linux/MacOS):
 
-Ubuntu/WSL/MacOS:
+```BASH
+docker run -p 4000:4000 -v $(pwd):/twofactorauth 2factorauth/twofactorauth
```

Comment:
Even when pulling `2factorauth/twofactorauth:latest`, I still get an error when running this command. It looks like the Dockerfile from #5546 is being used rather than the most recent one.

Reply:
The webp instructions are now gone